tests/test_umbral.py

```diff
@@ -1,9 +1,13 @@
 import pytest
 
 from umbral import umbral, keys
-
+import random
```

Comment:
Are you not using random anywhere? Can't seem to find it used here in this file.

Reply:
Ahh, yeah - at first I had forgotten that `gen_rand` existed on `Point` and `BigNum` so I was going to whip together one of my own.  Good catch.